rpc/cloudagent/cloud/availabilityzone/moc_cloudagent_availabilityzone.proto

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/cloudagent/cloud";
+package moc.cloudagent.availabilityzone;
+
+import "google/protobuf/wrappers.proto";
+import "moc_common_common.proto";
+
+message AvailabilityZoneRequest {
+	repeated AvailabilityZone AvailabilityZones = 1;
+	Operation OperationType = 2;
+}
+
+message AvailabilityZoneResponse {
+	repeated AvailabilityZone AvailabilityZones = 1;
+	google.protobuf.BoolValue Result = 2;
+	string  Error = 3;
+}
+
+message AvailabilityZone {
+    string name = 1;
+    string id = 2;
+    string locationName = 3;
+	repeated string nodes = 5;
```

Comment:
Please fix indentation

Reply:
![image](https://github.com/user-attachments/assets/c4276a70-68d9-4b96-9f47-1bae86794222)

This is showing fine in my local, not sure why it is showing differently here, tried multiple times fixing this but for some reason it is showing discrepancy between github and my local

Reply:
fixed the indentation part